release-notes/8.0/preview/api-diff/preview7/Microsoft.NETCore.App/8.0-preview7_System.Runtime.CompilerServices.md

```diff
@@ -0,0 +1,31 @@
+# System.Runtime.CompilerServices
```

Comment:
System.Runtime.CompilerServices LGTM. New attributes were exposed so Roslyn doesn't have to generate internal polyfills in every assembly.